com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -797,6 +798,11 @@ public override Material default2DMaterial
             get { return GetMaterial(DefaultMaterialType.Sprite); }
         }
 
+        public override Material default2DMaskMaterial
```

Comment:
Missing public API docs.

Reply:
Which docs are you referring to and where are they located?

Reply:
If you add public API you need to add public API docs in form of XML docs  https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/

You can find examples across the code. This generates API like this: https://docs.unity3d.com/Packages/com.unity.render-pipelines.universal@11.0/api/index.html